app/helpers/view_helper.rb

```diff
@@ -30,6 +30,10 @@ def link_to_devices_guidance_subpage(body, slug, html_options = {})
     govuk_link_to body, devices_guidance_subpage_path(subpage_slug: slug), html_options
   end
 
+  def breadcrumb(items)
```

Comment:
just one suggestion about the name of this: the singular 'breadcrumb' to me implies that this is for one component of the breadcrumbs/breadcrumb trail. Can I suggest calling it 'breadcrumbs' or 'breadcrumb_trail' instead?

Reply:
yep, good suggestion, fixed in 8d5acc7